addon_template/_dev_tools/defaults/formula_package/element.py

```diff
@@ -0,0 +1,67 @@
+import glob
+import os
+import pathlib
+import subprocess
+from typing import List
+
+from _dev_tools.addon_tasks.utils import (
+    CREDENTIALS_JSON_FILE,
+    save_json,
+    load_json,
+)
+
+FILE_EXTENSIONS = {".json"}
+EXCLUDED_FOLDERS = set()
+EXCLUDED_FILES = {"element.py"}
+
+
+def check_dependencies() -> None:
+    pass
+
+
+def bootstrap(
+        element_path: pathlib.Path,
+        url: str,
+        username: str,
+        password: str,
+        clean: bool,
+        global_python_env: pathlib.Path,
+        single_element: bool
+) -> None:
+    pass
+
+
+def get_build_dependencies() -> List[str]:
+    return []
+
+
+def build(element_path: pathlib.Path) -> None:
+    pass
+
+
+def deploy(element_path: pathlib.Path, url: str, username: str, password: str) -> None:
+    pass
```

Comment:
How does the formula package element get deployed?

Reply:
added a NotImplementedError. It will only showed if we try to deploy specifically the Formula Package, but it won't show if we deploy the entire Add-on package